apps/src/codebridge/types.ts

```diff
@@ -52,6 +52,7 @@ export type ConfigType = {
     };
   };
   activeGridLayout?: string;
+  showFileBrowser?: boolean;
```

Comment:
I'm wondering if we should split up this boolean into two values since it's overloaded (an `undefined` value means no button, but `true`/`false` means show or hide file browser itself, which tripped me up while looking at this).